atomsci/ddm/pipeline/perf_plots.py

```diff
@@ -818,6 +818,10 @@ def plot_prec_recall_curve(MP, epoch_label='best', plot_size=7, pdf_dir=None):
 
     """
     params = MP.params
+    if pdf_dir is not None:
+        pdf_path = os.path.join(pdf_dir, '%s_%s_model_%s_features_%s_split_ROC_curves.pdf' % (
+                                params.dataset_name, params.model_type, params.featurizer, params.splitter))
+        pdf = PdfPages(pdf_path)
```

Comment:
Where did these lines come from?

Reply:
Ok I think i understand this is providing the missing definitions for `pdf_path` and `pdf` 👍